frontend/src/components/pages/TranslationPage.tsx

```diff
@@ -184,35 +169,21 @@ const TranslationPage = () => {
 
       contentArray.sort((a, b) => a.lineIndex - b.lineIndex);
 
-      translatedContent.forEach(({ id, content, lineIndex }: Content) => {
-        const arrIndex = lineIndex - contentArray[0].lineIndex;
-        contentArray[arrIndex].translatedContent = content;
-        contentArray[arrIndex].storyTranslationContentId = id;
-      });
+      translatedContent.forEach(
+        ({ id, content, lineIndex, status }: Content) => {
+          const arrIndex = lineIndex - contentArray[0].lineIndex;
```

Comment:
Think this is unnecessary, without pagination `lineIndex === contentArray[0].lineIndex` always